kms/capi/ncrypt_windows.go

```diff
@@ -0,0 +1,572 @@
+//go:build windows
+// +build windows
+
+package capi
+
+import (
+	"bytes"
+	"crypto"
+	"crypto/elliptic"
+	"crypto/sha1"
+	"encoding/binary"
+	"errors"
+	"fmt"
+	"golang.org/x/sys/windows"
+	"syscall"
```

Comment:
I believe all calls to syscall can be just converted to windows, the original syscall is deprecated.

```go
windows.UTF16PtrFromString(s)
windows.Errno(0)
```